# Goal
Params params
Scenario scenario
Vehicle vehicle
---
# Final result
PathSolution solution
---
# Intermediate result
PathSolution solution
